### PR TITLE
index SW genre facet using same code that SearchWorks does

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

Part of #992;  this is the exemplar PR to allow remaining work to get done.

NOTE:  working on failing specs;  preparing a cocina-models PR to help

## How was this change tested? 🤨

Specs.

### CODE REVIEW

File: app/indexers/descriptive_metadata_indexer.rb [hunk 1]
```diff
@@ -15,7 +19,7 @@ def to_solr
       'sw_language_ssim' => language,
       'mods_typeOfResource_ssim' => resource_type,
       'sw_format_ssim' => sw_format,
-      'sw_genre_ssim' => display_genre,
```
Comment (justinlittman) [inline app/indexers/descriptive_metadata_indexer.rb:18]: Should the display_genre method be removed?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -90,7 +90,7 @@
   # order dependency and want to debug it, you can fix the order by providing
   # the seed, which is printed after each run.
   #     --seed 1234
-  config.order = :random
+  config.order = :random unless config.files_to_run.one?
```
Comment (justinlittman) [inline spec/spec_helper.rb:93]: What is this for?

**Review by justinlittman — APPROVED** [review]
Approve pending consideration of comments

File: spec/spec_helper.rb (hunk 2 above)
Comment (ndushay) [reply spec/spec_helper.rb:93]: so the specs run in order when I'm working with a single spec file ... it's easier to find the patterns of what is working and what isn't (e.g. if all the new ones are on the bottom).

File: app/indexers/descriptive_metadata_indexer.rb (hunk 1 above)
Comment (ndushay) [reply app/indexers/descriptive_metadata_indexer.rb:18]: yes thank you for the reminder!

**Review by aaron-collier — APPROVED** [review]

File: spec/indexers/descriptive_metadata/format_spec.rb [hunk 3]
```diff
@@ -1075,7 +1075,7 @@
       end
     end
 
-    context 'when groupedValue' do
+    context 'when groupedValue (which would be from MODS)' do
```
Comment (ndushay) [inline spec/indexers/descriptive_metadata/format_spec.rb:1078]: these changes are Arcadia approved!

File: spec/indexers/descriptive_metadata/pub_year_spec.rb [hunk 4]
```diff
@@ -143,11 +143,7 @@
         {
           title: [
             {
-              structuredValue: [
-                {
-                  value: 'Work & social justice'
-                }
-              ]
+              value: 'Work & social justice'
```
Comment (ndushay) [inline spec/indexers/descriptive_metadata/pub_year_spec.rb:146]: also Arcadia approved changes

**Pull request merged**